dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -132,4 +132,25 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1296
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Source>
+      <Package name="io.netty.util.internal.logging" />
+    </Source>
+    <Reason>
+      As io.netty.util.internal.logging.InternalLoggerFactory finds the available logging
```

Comment:
I know what you're trying to say, but I'm having trouble parsing this sentence. I suggest, "io.netty.util.internal.logging.InternalLoggerFactory catches and handles errors caused by the missing classes."

Reply:
Thank you. Updated.